config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: public.ecr.aws/j0a1m4z9/eks-anywhere-packages
+  newName: controller
```

Comment:
What is this change for? Can we add a comment next to it?

Reply:
This is Terry's personal public registry.